lib/rtcpeerconnection/chrome.js

```diff
@@ -117,12 +124,41 @@ function ChromeRTCPeerConnection(configuration) {
 
 inherits(ChromeRTCPeerConnection, EventTarget);
 
-ChromeRTCPeerConnection.prototype.addStream = function addStream(stream) {
-  if (this._localStreams.has(stream)) {
-    return;
+ChromeRTCPeerConnection.prototype.addTrack = function addTrack() {
+  var args = [].slice.call(arguments);
+  if ('addTrack' in PeerConnection.prototype) {
+    return this._peerConnection.addTrack.apply(this._peerConnection, args);
+  }
```

Comment:
We could potentially move this if-statement out, e.g.

```js
if (!('addTrack' in PeerConnection.prototype)) {
  ChromeRTCPeerConnection.prototype.addTrack = function addTrack() {
    // ...
  };
}
```

WDYT?

Reply:
@markandrus We don't follow this anywhere else. My first instinct was to do this, but then decided to go with the prevalent option.